### ISSUE TITLE
Unsupported block number error

### ISSUE THREAD

**Issue by rossneilson** [DXgovernance/dxvote#294]
I'm pretty sure this error has been around for a while but is only surfacing now because the loading screen now shows errors. Seems the UI is trying to check for events on a block that does not exist yet.
To replicate wait on a screen in the UI for a while, seems mostly random. 

Although we should fix this bug, since it currently interrupts the user but is not actually an urgent message, we should maybe just remove the error from the loading screen. 
If the fix is simple do it here, if not open a new issue once the error display is removed for this specific case. 

<img width="845" alt="スクリーンショット 2021-10-21 16 07 27" src="https://user-images.githubusercontent.com/39137239/138306818-0b1a49d0-cbde-4427-9395-a4931e087ea1.png">

**Comment by mprasanjith** [comment]
I also noticed a few other errors popping up like this which interrupts regular workflows. These needs to be fixed or handled better for sure, but will disable the error display for this for now.
